build/barys

```diff
@@ -396,9 +396,6 @@ else
     # Always activate resinhup bundles in production builds
     sed -i "s/.*RESINHUP ?=.*/RESINHUP ?= \"yes\"/g" conf/local.conf
 fi
-if [ -n "$SUPERVISOR_TAG" ]; then
-    sed -i "s/.*TARGET_TAG ?=.*/TARGET_TAG ?= \"$SUPERVISOR_TAG\"/g" conf/local.conf
-fi
```

Comment:
https://github.com/resin-os/resin-yocto-scripts/blob/master/build/barys#L419
https://github.com/resin-os/resin-yocto-scripts/blob/master/build/barys#L425

The variable in question is TARGET_TAG